man/systemd-id128.xml

```diff
@@ -80,7 +80,10 @@
     arguments are specified, they may be the identifiers or ID values of one or more known IDs, which are
     then printed with their name, or arbitrary IDs, which are then printed with a placeholder name. Combine
     with <option>--uuid</option> to list the IDs in UUID style, i.e. the way GPT partition type UUIDs are
-    usually shown.</para>
+    usually shown. This verb also shows a <literal>var-keyed</literal> entry, which, following the <ulink
+    url="https://uapi-group.org/specifications/specs/discoverable_partitions_specification">Discoverable
+    Partitions Specification</ulink>, is the GPT partition type UUID for <filename>/var/</filename>
```

Comment:
humm, no?

the /var/ thing is about the partition uuid, not the partition type uuid

Reply:
ok, then the description is unclear, as that's how I understood reading it

Reply:
actually, this change can still be useful, I'll just fix the doc